### PR TITLE
[PE- 1075] Update connection references

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

File: website/docs/d/connections.html.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+data "pureport_connections" "empty" {
+  account_href = "${data.pureport_accounts.main.accounts.0.href}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `network_href` - (Required) The HREF for the Pureport network associated with the connections.
```
Comment (Try75) [inline website/docs/d/connections.html.md]: I think this documented feature works with all Pureport resources, but the docs don't make that clear. It is also a little confusing to show Accounts in the "Example" and then "Networks" in the "Argument Reference".

File: website/docs/d/connections.html.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+- - -
+
+* `name_regex` - (Optional) A regex string to apply to the network list returned by Pureport. This
+  filtering is done locally on what Pureport returns, and could have a performance impact if the
+  result is large.
+
+## Attributes
+
+* `connections` - A list of Pureport connections.
```
Comment (Try75) [inline website/docs/d/connections.html.md]: ... then we talk about connections, so I'm really confused what this documentation is telling me.

File: website/docs/d/connections.html.md (hunk 1 above)
Comment (kevinkirkup) [reply website/docs/d/connections.html.md]: Corrected the Example above.
The account is required to get the Network.
The network is required for the connections.

File: website/docs/d/connections.html.md (hunk 2 above)
Comment (kevinkirkup) [reply website/docs/d/connections.html.md]: Connections here is what is returned after fetching data.
Arguments -> Input
Attributes -> Output

**Pull request merged**